src/streamlink/plugins/zdf_mediathek.py

```diff
@@ -128,15 +128,16 @@ def _get_streams(self):
             return
 
         headers = {
+            "Accept": "application/vnd.de.zdf.v1.0+json",
             "Api-Auth": "Bearer {0}".format(zdf_json['apiToken']),
             "Referer": self.url
         }
 
         res = self.session.http.get(zdf_json['content'], headers=headers)
         document = self.session.http.json(res, schema=_documents_schema)
 
-        stream_request_url = document["mainVideoContent"]["http://zdf.de/rels/target"]["http://zdf.de/rels/streams/ptmd"]
-        stream_request_url = API_URL + stream_request_url
+        stream_request_url = document["mainVideoContent"]["http://zdf.de/rels/target"]["http://zdf.de/rels/streams/ptmd-template"]
```

Comment:
Kika has an invalid space at the end,
https://www.zdf.de/sender/kika/kika-live-beitrag-100.html

```
'/tmd/2/{playerId}/live/ptmd/247onAir-15 '
```

streamlink should probably remove it